packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -125,7 +125,11 @@ class ModalLauncherKeypressListener extends React.Component<{
     }
 
     _handleKeyup = (e: KeyboardEvent) => {
-        if (e.key === "Escape") {
+        const escape = 27;
+        const keyCode = e.which || e.keyCode;
+        if (keyCode === escape) {
+            e.preventDefault();
+            e.stopImmediatePropagation();
```

Comment:
Can there be a situation where a tooltip is still active below a modal that's been opened?  I'm guessing that this is defensive.  Can you add a comment about the intent?

Reply:
Not with the changes that make tooltips one at a time. And yes, I can add a comment.